src/extension.ts

```diff
@@ -391,77 +392,81 @@ export async function activate(context: vscode.ExtensionContext) {
     binTimestampEventFunc
   );
   context.subscriptions.push(buildWatcherCreateDisposable);
-
```

Comment:
Changes from 394 to 469 are not related to the PR proposal solution. While I think is good to add this to `context subscription` disposable array, the amount of changes are not necessary. 

Please update to remove these changes.

Reply:
Yes, this was the issue I had with the rebase. I think I might have change this part of the code in the past with another rebase. Here is where I've tried to make manual fixes on the past rebase.

I will try to back on past rebases on my local to see if I can fix this the proper way

Reply:
I finally figured it out, the problem is with this commit: https://github.com/espressif/vscode-esp-idf-extension/commit/5de32524f8abfabd59b594aca33b51b88c2716f1

I will revert one commit before and will try to cherry picks the other commits or add the changes from them manually

Reply:
Hi @brianignacio5,

I've done what I've mentioned above, please take another look